chirpr.py

```diff
@@ -1,4 +1,4 @@
-from flask import Flask, request, g, render_template, redirect, url_for, flash, session
+from flask import Flask, request, g, render_template, redirect, url_for, flash, session, Response
```

Comment:
I'm curious how you are using it. You can keep it for now, but we might be able to find a better way to test.